dwave/cloud/auth/creds.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from pathlib import Path
+from typing import List, Optional, Type, Union
+
+import diskcache
+
+from dwave.cloud.config import get_configfile_paths, get_default_configfile_path
+
+__all__ = ['Credentials']
+
+logger = logging.getLogger(__name__)
+
+CREDS_FILENAME = diskcache.core.DBNAME = "credentials.db"
```

Comment:
Wouldn't this interfere with another package also using `diskcache` by globally overloading the database name?

Reply:
It would. Not just another package, but it will affect our own `@cached.ondisk`. I will isolate this better. Unfortunately, there's no easy/nice way to isolate it..

Reply:
Pushed an inelegant update (subclassed `diskcache.Cache`), but the only one I could see working robustly.

Reply:
I'll submit a PR upstream with this change, and then we can remove it from Ocean.